doc/stdenv/stdenv.chapter.md

```diff
@@ -77,7 +77,7 @@ where the builder can do anything it wants, but typically starts with
 source $stdenv/setup
 ```
 
-to let `stdenv` set up the environment (e.g., process the `buildInputs`). If you want, you can still use `stdenv`’s generic builder:
+to let `stdenv` set up the environment (e.g., set an empty `PATH`, process the `buildInputs`). If you want, you can still use `stdenv`’s generic builder:
```

Comment:
This makes it sound too much like something else populates the `PATH` IMO.

Reply:
```suggestion
to let `stdenv` set up the environment (e.g. by resetting `PATH` and populating it from `buildInputs`). If you want, you can still use `stdenv`’s generic builder:
```
Better?